packages/compiler/src/template_parser/binding_parser.ts

```diff
@@ -49,14 +49,24 @@ export class BindingParser {
 
   getUsedPipes(): CompilePipeSummary[] { return Array.from(this._usedPipes.values()); }
 
-  createBoundHostProperties(dirMeta: CompileDirectiveSummary, sourceSpan: ParseSourceSpan):
-      ParsedProperty[]|null {
+  createBoundHostProperties(
+      dirMeta: CompileDirectiveSummary, sourceSpan: ParseSourceSpan,
+      absoluteOffset?: number): ParsedProperty[]|null {
+    if (!absoluteOffset) {
```

Comment:
```suggestion
    if (absoluteOffset === undefined) {
```

Especially because `absoluteOffset` is of type `number`, so we shouldn't mess with 0.